packages/base/room.gts

```diff
@@ -230,9 +234,11 @@ const roomStateCache = new WeakMap<RoomCard, RoomState>();
 
 export class RoomCard extends Card {
   // This can be used  to get the version of `cardInstance` like:
-  //   Reflect.getProtypeOf(roomCardInstance).constructor.getVersion(cardInstance);
+  // Reflect.getProtypeOf(roomCardInstance).constructor.getVersion(cardInstance);
+  // Note: this does make a SERIOUS assumption about time
+  // Events from different homeservers may be different
   static getVersion(card: Card) {
-    return cardVersions.get(card);
+    return cardVersions.get(card)?.serverTimeOrigin;
```

Comment:
This is an unrealiable way to track versions if clients are connected to different homeservers but its ok for now